packages/react-components/src/molecules/Collapsible.tsx

```diff
@@ -27,7 +47,7 @@ const Collapsible: React.FC<CollapsibleProps> = ({
           >
             {expanded ? chevronCircleUpIcon : chevronCircleDownIcon}
           </span>
-          {expanded ? 'Hide' : 'Show'}
+          {expanded ? 'Hide' : 'Show'} more
```

Comment:
<img width="1003" alt="image" src="https://user-images.githubusercontent.com/9073485/159233733-bb29f72a-6a04-4a3f-9046-77488d22e3f7.png">

Reply:
<img width="981" alt="image" src="https://user-images.githubusercontent.com/9073485/159233795-2def6b87-8ea0-46be-8aab-76390c8418d5.png">

Reply:
I guess that it will be hard to make it "smarter"

Reply:
Yeah, I don't really like it which is why i got beatriz to acknowledge and accept the problems in refinement. I think it be better to show the text in full or ask for an additional summary field.